### PR TITLE
cocoapods update

### PR DESCRIPTION
**Pull request by ndis1**
cocoapods update. doing this as a separate PR because it's big and distracting

### CODE REVIEW

**Comment by flippinjoe** [comment]
I think while we're  updating this we should remove the FBSnapshot shiz.   We never used it, and if we  do add UITesting the built in would be preferred

File: Example/Pods/Local Podspecs/URBNSwipeableController.podspec.json [hunk 1]
```diff
@@ -1,25 +1,19 @@
 {
   "name": "URBNSwipeableController",
-  "version": "0.1.0",
-  "summary": "A short description of URBNSwipeableController.",
-  "description": "                       An optional longer description of URBNSwipeableController\n\n                       * Markdown format.\n                       * Don't worry about the indent, we strip it!\n",
-  "homepage": "https://github.com/<GITHUB_USERNAME>/URBNSwipeableController",
+  "version": "0.2.4",
+  "summary": "URBNSwipeableController allows you to add swiping to any tableviewcell, collectionviewcell, or any old view.",
+  "homepage": "https://github.com/urbn/URBNDataSource",
```
Comment (flippinjoe) [inline Example/Pods/Local Podspecs/URBNSwipeableController.podspec.json:5]: Wrong homepage

**Comment by flippinjoe** [comment]
Also, do you need a special travis.yml file for cocoa pods 1.0 since this is public?

**Comment by ndis1** [comment]
@flippinjoe21 I would if there were a travis.yml in this project 😉

**Pull request merged**